schemas/common.json

```diff
@@ -718,6 +718,85 @@
             "items": {
                 "type": "string"
             }
+        },
```

Comment:
Minor things here. In the health reports configuration, we specify that the configuration applies generally to `components` and the settings are specific to SMTP, such as:

```
# < Health Report SMTP Server >
# Address of SMTP server to be used if emailing health results
#
# health_report_email_server: ""

# < Health Report SMTP Server Port >
# Port to be used on the SMTP Server
#
# health_report_email_server_port: 25
```

Similarly, here we should identify that monit is coming from the VSD and that the configuration is for SMTP. No need to change the variable names, but the titles and descriptions should say something like:

```
<VSD Monit SMTP Server>
<VSD Monit SMTP Server Port>
```